### DISCUSSION TITLE
Is it possible to put a fixed value in the body?

### DISCUSSION THREAD

**Discussion by linwaytin** [tarsius/llama#6] [Support] [answered]
It is usual for me to run a function taking fixed arguments with hooks, like
```elisp
 (add-hook 'evil-insert-state-entry-hook '(lambda () (hl-line-mode 1)))
```
This is not bad, but I think the `lambda ()` is kind of unnecessary, as the main purpose here is to run the function with an argument. 

Can `llama` help with this issue? Thanks.

**Comment by meedstrom** [comment, marked as answer]
Yes, these are equivalent:

```elisp
(add-hook 'evil-insert-state-entry-hook (lambda () (hl-line-mode 1)))
(add-hook 'evil-insert-state-entry-hook (##hl-line-mode 1))
```

Enjoy!

(note lambdas are self-quoting, so there is no need to put a quote sign out front)

**Comment by linwaytin** [comment]
Great! Thanks a lot. 

I think this macro should be built-in in elisp. The readability is increased a lot!
